tokens/css-utilities.js

```diff
@@ -1,21 +1,18 @@
-const chroma = require('chroma-js');
-const { darken } = require('./sass-helpers');
-
 /**
  * Implements bg-variant mixin from bootstrap. Creates utility classes for background colors based on theme color.
  */
 function bgVariant(token) {
-  const { attributes: { type, item }, value } = token;
+  const { attributes: { type, item }, actions } = token;
   const parent = `.bg-${type}${item === 'base' ? '' : `-${item}`}`;
   return `${parent} {
-  background-color: ${value} !important;
+  background-color: ${`var(--${token.name})`} !important;
```

Comment:
combined with the previous suggestion we could then just have

```suggestion
  background-color: ${`var(--${name})`} !important;
```